internal/statestore/redis.go

```diff
@@ -535,36 +516,45 @@ func (rb *redisBackend) UpdateAssignments(ctx context.Context, ids []string, ass
 	if assignment == nil {
 		return status.Error(codes.InvalidArgument, "assignment is nil")
 	}
+	if len(ids) == 0 {
+		return nil
+	}
 
 	redisConn, err := rb.connect(ctx)
 	if err != nil {
 		return err
 	}
 	defer handleConnectionClose(&redisConn)
 
-	err = redisConn.Send("MULTI")
+	b := make([]interface{}, len(ids))
+	for i := range ids {
+		b[i] = ids[i]
+	}
+
+	ticketBytes, err := redis.ByteSlices(redisConn.Do("MGET", b...))
```

Comment:
How does this surface error with missing Tickets? Can we isolate that error from any other redis error? I ask because we have issue https://github.com/googleforgames/open-match/issues/1121 which most certainly need us to ignore 'Not Found' failures (not just ignore - but idenitfy the IDs for these and pass them back in some status object) so that makes it critical to identify and ignore the not found errors from the rest. Possible to do that here?